### ISSUE TITLE
About the first formal release version

### ISSUE THREAD

**Issue by zhenfei0601** [eclipse-cyclonedds/cyclonedds#123]
I asked this question before this year. 
At that time I got the reply that the first formal release version would be released out before the March 1st.
Now I want to know if I can got the release version before tomorrow.
Thank you!

**Comment by eboasson** [comment]
Hi @zhenfei0601, it hasn't been forgotten. I did hedge a bit by saying I would strive for March 1 to have some room for slippage and still get it done in the Q1 as was your original request.

As it happens, that slippage did occur because the release review documentation I filed raised some questions from the Eclipse Foundation's Program Management Committee, causing me to miss the February 20 release review date that I aimed for. I hope I have just now resolved the last question and that it will be approved for release on the March 6 review by the Eclipse Management Organization.
